R/read_data.R

```diff
@@ -113,7 +113,12 @@ read_data <- function(data_path,
       cli::cli_abort(
         c(
           "Error fetching data from {.path {data_path}}",
-          "Using parameters {parameters}",
+          "Using parameters:",
+          "*" = "data_path: {.path {parameters[[1]]}}",
```

Comment:
Numbers here vs. `[["data_path"]]`?

Reply:
Some utility which does this automatically? i.e. don't have to change it if `parameters` changes

Reply:
https://github.com/CDCgov/cfa-epinow2-pipeline/pull/79/commits/852e07a5b135e70332f209686972dbfa96db9518